src/acore-wp-plugin/src/Hooks/User/ConnectionLogger.php

```
@@ -0,0 +1,362 @@
+        KEY user_id (user_id),
+        KEY login_at (login_at)
+    ) $collate;";
+
+    require_once ABSPATH . 'wp-admin/includes/upgrade.php';
+    dbDelta($sql);
+    update_option('acore_login_history_db_version', '2');
+}
+
+add_action('wp_login', __NAMESPACE__ . '\\acore_log_login', 10, 2);
+
+function acore_log_login($user_login, $user) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user_login'. (undefined)

(UnusedFormalParameter)